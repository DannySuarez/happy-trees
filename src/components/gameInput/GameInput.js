import React from 'react';
import PropTypes from 'prop-types';
import styles from './gameinput.css';

function GameInput({ guesses, guess, handleSubmit, handleChange  }) {
  return (
    
    <footer>
      <section>
        <aside>
          <p>You have {guesses} 3 guesses left</p>
        </aside>
      </section>

      {/* <section>
        <aside className="rounds">
          <p>Round 1{} of 5</p>
        </aside>
      </section> */}
    
      <form className={styles.form} onSubmit={handleSubmit}>
        <input className={styles.input} type="text" name="guess" value={guess} onChange={handleChange} />
        <button>+</button>
      </form>
    </footer>
    
  );

}

GameInput.propTypes = {
  guesses: PropTypes.string,
  guess: PropTypes.string,
  handleSubmit: PropTypes.func,
  handleChange: PropTypes.func
};

export default GameInput;
