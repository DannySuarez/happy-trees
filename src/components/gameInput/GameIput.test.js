import React from 'react';
import { shallow } from 'enzyme';
import GameInput from './GameInput';

describe('Game Input', () => {
  it('renders Game input component', () => {
    const wrapper = shallow(<GameInput />);
    expect(wrapper).toMatchSnapshot();
  });
});
