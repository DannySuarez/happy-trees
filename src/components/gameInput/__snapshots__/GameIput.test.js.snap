// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game Input renders Game input component 1`] = `
<footer>
  <section>
    <aside>
      <p>
        You have 
         3 guesses left
      </p>
    </aside>
  </section>
  <section>
    <aside
      className="rounds"
    >
      <p>
        Round 1
         of 5
      </p>
    </aside>
  </section>
  <form
    className="form"
  >
    <input
      className="input"
      name="guess"
      type="text"
    />
  </form>
</footer>
`;
